### PR TITLE
feat(search): configure CORS

### PR DESCRIPTION
**Pull request by sinedied**
~~I need to deploy it first to test it~~

### CODE REVIEW

**Comment by sinedied** [comment]
Tested and validated on deployed version

File: infra/main.parameters.json [hunk 1]
```diff
@@ -68,6 +68,9 @@
     "useApplicationInsights": {
       "value": "${AZURE_USE_APPLICATION_INSIGHTS=false}"
     },
+    "allowedOrigin": {
+      "value": "${ALLOWED_ORIGIN}"
+    },
```
Comment (anfibiacreativa) [inline infra/main.parameters.json:73]: Output is ALLOWED_ORIGINS and value is ALLOWED_ORIGIN. Is that a typo?

**Review by anfibiacreativa — CHANGES_REQUESTED** [review]
Please take a look at mismatching env var name. Is that an iterator? ALLOWED_ORIGIN of ALLOWED_ORIGINS? Otherwise seems to be a typo.

File: infra/main.parameters.json (hunk 1 above)
Comment (sinedied) [reply infra/main.parameters.json:73]: No, that's wanted: `ALLOWED_ORIGIN` is one additional frontend url (for future separate deployments), and output `ALLOWED_ORIGINS` is the list of ALL allowed origins, including the optional input one and our frontend

**Comment by sinedied** [comment]
@anfibiacreativa I answered your comment, it's not a mismatch it's intentional as one is the input (single), and one is the output (plural) consumed by the search service.

**Comment by manekinekko** [comment]
LGTM

**Pull request merged**